src/customFileParser/unpackGeometry/InstancedMesh.ts

```diff
@@ -37,22 +37,19 @@ export default function unpackInstancedMeshes(
       geometryInfo.indices.rewind();
       geometryInfo.nodeIds.rewind();
     }
-  }
 
-  // Create mesh collections for each file Id and triangle offset
-  const collections: {[fileId: string]: {[triangleOffset: string]: InstancedMeshCollection}} = {};
-  Object.keys(meshCounts).forEach(fileId => {
-    collections[fileId] = collections[fileId] ? collections[fileId] : {};
-    Object.keys(meshCounts[fileId]).forEach(triangleOffset => {
-      const { count, triangleCount } = meshCounts[fileId][triangleOffset];
-      collections[fileId][triangleOffset] = new InstancedMeshCollection(
-        parseInt(triangleOffset, 10), triangleCount, count);
+    // Create mesh collections for each file Id and triangle offset
+    const collections: {[fileId: string]: {[triangleOffset: string]: InstancedMeshCollection}} = {};
+    Object.keys(meshCounts).forEach(fileId => {
+      collections[fileId] = collections[fileId] ? collections[fileId] : {};
```

Comment:
I like being explicit here with a check with `undefined` or `null`, and not any truthy value.

Reply:
Fixed